charts/gardener/gardenlet/charts/runtime/templates/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
 {{ toYaml .Values.global.gardenlet.podLabels | indent 8 }}
         {{- end }}
     spec:
-      priorityClassName: gardenlet
+      priorityClassName: gardener-system-critical-mig
```

Comment:
why not use `gardener-system-critical-migration` fully?

Reply:
I am fine with anything but `gardener-system-critical-mig` is small.

Reply:
Please use `gardener-system-critical-migration`